README.md

```diff
@@ -43,93 +43,94 @@ pip install dash-tools
 
 ## **Usage Examples**
 
-Below are common usage examples. For a more in-depth tutorial on writing apps for Plotly Dash, see the [Plotly Dash Documentation](https://dash.plotly.com/layout). For information about dash-tools commands, read the [_Commands_](#commands) section.
+### A. Create an App
 
-### **Creating A New Project**
-
-> _For a more in-depth tutorial on using dash-tools, check out the [Create and Upload Your App](/docs/Configuring-Your-App.md) document!_
-
-Creating a new Dash project is very simple. The following command will create a new directory called "MyDashApp":
+1. Create a Dash project in a new directory called "MyDashApp" (using your terminal or command prompt):
 
 ```bash
 dash-tools --init MyDashApp
 ```
 
-Optionally, templates can be used. Check out [Templates](#templates) for more details.
-
-You can see what files are included with your new app:
+2. Open the default app:
 
 ```bash
-cd MyDashApp
+.\MyDashApp\src\app.py
 ```
 
-You can make changes to your app in the `src/app.py` file! See [Plotly Dash Layout Docs](https://dash.plotly.com/layout) for more information, or check out the dash-tools docs on [Configuring Your Dash App](/docs/Configuring-Your-App.md).
+3. Replace the code in `app.py` with your own app code. Make sure to keep the `server = app.server` right after app instantiation:
+
+![update-app](https://user-images.githubusercontent.com/32049495/169304171-bf23b2d0-26b4-4767-b38f-cd6586ddf56e.gif)
 
-When you are happy with your changes, run your dash app locally with the following command. You will be able to view your app at http://127.0.0.1:8050/ in your browser:
+4. Run your app to ensure it works:
 
 ```bash
-python src/app.py
+python .\MyDashApp\src\app.py
```

Comment:
This command will only work on windows, as Linux and Mac OS uses forward slashes "/" to denote directories.

Should we add a second code block for Linux and Mac here that uses the forward slashes?